crates/router/src/routes/app.rs

```diff
@@ -1271,6 +1271,10 @@ impl PaymentLink {
                 web::resource("{merchant_id}/{payment_id}")
                     .route(web::get().to(initiate_payment_link)),
             )
+            .service(
+                web::resource("s/{merchant_id}/{payment_id}")
```

Comment:
Was this supposed to be only `/{merchant_id}/{payment_id}`?

Reply:
We've decided to add `/s/` for secure links

Reply:
we should change it to `secure`

Reply:
This was added based on the discussions, primarily for ensuring the context of open / secure links can not be inferred from these links // @knutties